src/Adapter/MSTest.TestAdapter/buildTransitive/common/MSTest.TestAdapter.targets

```diff
@@ -1,4 +1,10 @@
-<Project>
+<Project InitialTargets="NETStandardCompatError_MSTest_TestAdapter">
+
+    <Target Name="NETStandardCompatError_MSTest_TestAdapter"
+            Condition="'$(SuppressTfmSupportBuildWarnings)' == ''">
+        <Warning Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard'"
+                 Text="MSTest.TestAdapter doesn't support $(TargetFramework) and has not been tested with it. Note that test libraries shouldn't reference MSTest.TestAdapter or MSTest metapackage. Instead, consider referencing only MSTest.TestFramework. You may also set &lt;SuppressTfmSupportBuildWarnings&gt;true&lt;/SuppressTfmSupportBuildWarnings&gt; in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk." />
```

Comment:
What about something like:

MSTest.TestAdapter does not support the target framework '$(TargetFramework)' and has not been tested with it. Test libraries (projects that don't directly run tests) should not reference MSTest.TestAdapter or the MSTest metapackage. Instead, reference only MSTest.TestFramework. To suppress this warning and proceed at your own risk, add &lt;SuppressTfmSupportBuildWarnings&gt;true&lt;/SuppressTfmSupportBuildWarnings&gt; to your project file.

Reply:
```suggestion
                 Text="MSTest.TestAdapter does not support the target framework '$(TargetFramework)' and has not been tested with it. Test libraries (projects that don't directly run tests) should not reference MSTest.TestAdapter or the MSTest metapackage. Instead, reference only MSTest.TestFramework. To suppress this warning and proceed at your own risk, add &lt;SuppressTfmSupportBuildWarnings&gt;true&lt;/SuppressTfmSupportBuildWarnings&gt; to your project file." />
```